docs/src/content/en/reference/agents/_meta.ts

```diff
@@ -1,15 +1,23 @@
 const meta = {
   agent: "Agent",
   getAgent: "getAgent()",
-  createTool: "createTool()",
   generate: "generate()",
   stream: "stream()",
+  streamVNext: "streamVNext()",
   getWorkflows: "getWorkflows()",
-  getVoice: "getVoice()",
-  getInstructions: "getInstructions()",
   getTools: "getTools()",
+  getScorers: "getScorers()",
   getModel: "getModel()",
   getMemory: "getMemory()",
+  getVoice: "getVoice()",
+  getDescription: "getDescription()",
+  getInstructions: "getInstructions()",
+  getLLM: "getLLM()",
+  hasOwnMemory: "hasOwnMemory()",
```

Comment:
we should skip this one, it's a public method, but not really intended for users to call

Reply:
my bad, I mean "hasOwnMemory"